### PR TITLE
Brickpi docs update (input port 5)

### PR DESCRIPTION
**Pull request by bmegli**
Added page for BrickPi input port 5.
Added mention and link in ports (it's not a driver but this is where I would look for it).

As a bonus corrected typo...

### CODE REVIEW

File: docs/ports/index.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ sensor multiplexers fall into this category (see list below). Simple
 multiplexers that only support one connection type will not have a separate
 driver for the port.
 
+BrickPi Input Port 5 is RaspberryPi I2C physically wired to the port pins. See <a href="brickpi-in-port-5">brickpi-in-port-5</a> for the details.
```
Comment (dlech) [inline docs/ports/index.md]: > BrickPi Input Port 5 is RaspberryPi I2C physically wired to the port pins.

This sentence is not very clear. Maybe "Input Port 5 on the BrickPi is pyhsically wired to the I2C pins on the Raspberry Pi".

Also, markdown does not need html tags for the link. Replace `<a href="brickpi-in-port-5">brickpi-in-port-5</a>` with `[brickpi-in-port-5]` and also `[brickpi-in-port-5]: /docs/ports/brickpi-in-port-5` on a new line.

File: docs/ports/index.md [hunk 2]
```diff
@@ -24,7 +26,7 @@ kernel.
     <th>Description</th>
     <th>Connection Types</th>
     <th>Module</th>
-    </tr>
+    </tr>
```
Comment (dlech) [inline docs/ports/index.md]: no need to add trailing whitespace

**Comment by bmegli** [comment]
ping, added suggested fixes some links in brickpi-in-port-5 page and clarified which sensors are supported on this port

**Comment by bmegli** [comment]
+ some minor changes to improve readability in brickpi-in-port-5 page

**Comment by bmegli** [comment]
Ugh, added back the sentence and link to using-i2c-sensors that got lost somehow ;-)
Painful.

**Pull request merged**